enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,725 @@
+Edges with matching blocks are conditionally recommended, and will be included in `conditionalEdges`.
+
+### Cluster-version operator support for the enhanced schema
+
+The cluster-version operator will learn to parse [`conditionalEdges`](#enhanced-cincinnati-json-representation) into [`conditionalUpdates`](#enhanced-clusterversion-representation).
+`edges` will continue to go straight into `availableUpdates`.
+The operator will log an error if the same target is included in both `edges` and `conditionalEdges`, but will prefer the `conditionalEdges` entry in that case.
+
+Additionally, the operator will continually re-evaluate the blocking conditionals in `conditionalUpdates` and update `conditionalUpdates[].conditions` accordingly.
+The timing of the evaluation and fresheness are largely internal details.
+The operator can periodically poll, or use edge triggered watches, combinations of watching and polling, etc.
+
+To perform the PromQL request, the operator will FIXME: details about connecting to the local Thanos/Prometheus.
+
+If there are issues evaluating a conditional update, the operator will set the `Unknown` status on the `Recommended` condition.
+The operator will grow a new `warning`-level `CannotEvaluateConditionalUpdates` alert that fires if `lastTransitionTime` for any `Recommended=Unknown` condition is over an hour old.
+
+Any `conditionalUpdates` with `Recommended=True` will have its release inserted into `availableUpdates`.
+
+Both `availableUpdates` and `conditionalUpdates` should be sorted in decreasing SemVer order for stability, to avoid unnecessary status churn.
+
+The cluster version operator does not currently gate update acceptance on whether the requested target release is a recommended update.
+With this enhancement, the CVO will add non-blocking gates.
+Update targets that are not currently recommended, or are not supported at all, will be allowed without any ClusterVersion overrides ([`force`][api-force] or a similar, new property).
+But update targets that are not currently recommended will result in entries in [the `overrides` history entries](#enhanced-clusterversion-representation).
+
+### Update client support for the enhanced schema
+
+[The web-console][web-console] and [`oc`][oc] both consume ClusterVersion to present clients with a list of available updates.
+With this enhancement, they will both be extended to consume [`conditionalUpdates`](#enhanced-clusterversion-representation).
+When listing recommended updates, clients will list the contents of `availableUpdates`.
+When listing all supported updates, clients will additionally include entries from `conditionalUpdates` with `Recommended!=True`, and include the `reason` and `message` from the `Recommended` condition alongside the supported-but-not-recommended updates (if this output is too verbose, clients may want to hide it behind an option like `--include-not-recommended`).
```

Comment:
We'e had a reference to the docs for a while now:

```console
$ git show | grep -A1 Note
+Note: as described [in the documentation][support-documentation], supported updates are still supported even if incoming edges are blocked, and Red Hat will eventually provide supported update paths from any supported release to the latest supported release in its z-stream.
+And, since [docs#32091][openshift-docs-32091], [that documentation][support-documentation] also points out that updates initiated after the update recommendation has been removed are still supported.
```

And I've just pushed 66999b1 -> 0f8c314, adding a new *Comparison with tombstones* subsection under *User Stories* to talk through @sdodson's points above.